src/yetibot/core/models/admin.clj

```diff
@@ -11,8 +11,36 @@
 
 (defn config [] (get-config ::config [:admin]))
 
-(defn admin-only-command? [cmd]
-  (boolean ((-> (config) :value :commands set) cmd)))
+(comment
+  (config)
+  )
 
-(defn user-is-admin? [{:keys [id]}]
-  (boolean ((-> (config) :value :users set) id)))
+(defn admin-only-command?
+  "See if cmd is in the list of admin commands as defined
+   by the instance 'config'. Arity/2 allows for passing in
+   custom config to compare against, mainly used for testing."
```

Comment:
wow, interesting that `cljfmt` made docstrings crazy narrow! I kind of like it though.